src/should-forward-property.js

```diff
@@ -600,17 +26,18 @@ const isCustomAttribute = RegExp.prototype.test.bind(
 )
 
 const hasItem = (list, name) => list.indexOf(name) !== -1
-const isHtmlProp = name => hasItem(htmlProps, name)
+const isHtmlProp = (name, tagName) => {
+  const elementAttributes = reactHTMLAttributes[tagName]
+
+  return hasItem([...globalReactHtmlProps, ...elementAttributes], name)
```

Comment:
@kentcdodds 

I will implement those. And I have some other ideas that may make this faster. 

Will putting `isReactProp(name)` before `isCustomAttribute(name)` make it faster? Because I think the former could be a more common use case. On a related note, how about putting `!isCssProp(name)` before `tagName === 'svg'`? Since I think the former may turn out to be true more often than the latter. 

Another idea is caching the result of `shouldForwardProperty` in its file by using closure so that the combination of `tagName` and `name` that has already been checked before doesn't need to go through the same process again to get the result.